clusterloader2/testing/huge-service/config.yaml

```diff
@@ -0,0 +1,33 @@
+# Huge service test config
+{{$HUGE_SERVICE_HEADLESS := DefaultParam .CL2_HUGE_SERVICE_HEADLESS false}}
+{{$HUGE_SERVICE_ENDPOINTS := DefaultParam .CL2_HUGE_SERVICE_ENDPOINTS 1000}}
+
+name: huge-service
+namespace:
+  number: 1
+tuningSets:
+- name: Sequence
+  parallelismLimitedLoad:
+    parallelismLimit: 1
+- name: HugeServiceParallel
+  parallelismLimitedLoad:
+    parallelismLimit: 1
+- name: DeletionTimeLimited
+  TimeLimitedLoad:
+    timeLimit: 5m
+steps:
+- module:
+    path: modules/measurements.yaml
+    params:
+      action: start
+- module:
+    path: modules/service.yaml
+    params:
+      endpoints: {{$HUGE_SERVICE_ENDPOINTS}}
+      isHeadless: {{$HUGE_SERVICE_HEADLESS}}
+      namespaces: 1
```

Comment:
Do we need to make namespaces configurable? I'm afraid setting value significantly larger than 1 may trigger some timeouts on pod deletion.

Reply:
I had the same question in my head when writing this and decided to retain similarity with scheduler-throughput. But I guess if you don't see the value in having it then it can be safely removed. Fixed.